src/main/java/cz/kominekjan/disenchantment/Disenchantment.java

```diff
@@ -54,9 +55,13 @@ public void onEnable() {
         File configFile = new File(plugin.getDataFolder(), "config.yml");
 
         try {
-            ConfigUpdater.update(plugin, "config.yml", configFile);
+            ConfigUpdater.update(plugin, "config.yml", configFile,
+                    "enchantments-status", "book-splitting-enchantments-status",
+                    "disabled-enchantments", "disabled-book-splitting-enchantments");
```

Comment:
That is the part where I have issue.
I do not really know the use of ConfigUpdater. Could you clarify ?

But from my testing:
If not set are ignored values, then it will remove some of the configurations.
If set as ignored values, then It may throw a Runtime Exception if the value is not present.

A better understanding of the reason ConfigUpdater is used would clarify what should we do.

Reply:
The config updater is a util for smart config updates e.g. when adding a new key it does add it to the current configuration without any collisions or replacements of the old config keys, so it makes updating the config easier (as I look at it now, it does not :D What I would probably do that I would leave the old keys in the config and add the new ones, cause I am not sure if the config updater removes the old keys if you remove then in the new configuration)

for more info there is git repo for this config I am using: https://github.com/tchristofferson/Config-Updater